qa/L0_pytorch_unittest/test.sh

```diff
@@ -10,6 +10,7 @@ pip install pytest==6.2.5 onnxruntime==1.13.1
 pytest -v -s $TE_PATH/tests/pytorch/test_sanity.py
 PYTORCH_JIT=0 NVTE_ALLOW_NONDETERMINISTIC_ALGO=0 pytest -v -s $TE_PATH/tests/pytorch/test_numerics.py
 pytest -v -s $TE_PATH/tests/pytorch/test_jit.py
-pytest -v -s $TE_PATH/tests/pytorch/test_fused_attn.py
+pytest -v -s $TE_PATH/tests/pytorch/fused_attn/test_fused_attn.py
+pytest -v -s $TE_PATH/tests/pytorch/fused_attn/test_fused_attn_with_cp.py
```

Comment:
Just a headsup: I will have to move L14 to a different test directory (qa/L0_pytorch_distributed_unittest/test.sh) since it requires more than 1 GPU. I can make the change and push to your branch.

Reply:
sure, sounds good. thanks.